src/pages/storage/forms/StoragePoolFormMain.tsx

```diff
@@ -46,6 +48,12 @@ const StoragePoolFormMain: FC<Props> = ({ formik }) => {
   const isPowerFlexDriver = formik.values.driver === powerFlex;
   const isPureDriver = formik.values.driver === pureStorage;
   const storageDriverOptions = getStorageDriverOptions(settings);
+  const hasClusterWideSource = isCephDriver || isCephFSDriver;
+  const hasSource = !isPureDriver && !isPowerFlexDriver;
+
+  const helpText = formik.values.isCreating
+    ? getSourceHelpForDriver(formik.values.driver)
```

Comment:
Please update the `getSourceHelpForDriver` function for btrfs, as we enabled it in this PR, the current help text needs to be adjusted:

![image](https://github.com/user-attachments/assets/450a7174-1ec1-4f38-982d-1218137d9eaf)